renovate.json

```diff
@@ -7,5 +7,13 @@
     "automerge": false,
     "schedule": [
         "every weekend"
+    ],
+    "regexManagers": [
+        {
+          "fileMatch": ["^Dockerfile$"],
+          "matchStrings": ["ENV ASDF_VERSION=(?<currentValue>.*?)\\n"],
+          "depNameTemplate": "asdf-vm/asdf",
+          "datasourceTemplate": "github-releases"
+        }
```

Comment:
What is this part for?

Reply:
Picking up the asdf version in the Dockerfile. As soon as there is a new asdf version, renovate will pick it up and update the Dockerfile.